Sources/OpenLogin/WebAuth.swift

```diff
@@ -2,21 +2,65 @@ import UIKit
 import AuthenticationServices
 import SafariServices
 
+public struct OLInitParams {
+    public init(clientId: String, network: Network, sdkURL: URL? = nil) {
+        self.clientId = clientId
+        self.network = network
+        if let sdkURL = sdkURL {
+            self.sdkURL = sdkURL
+        }
+    }
+    
+    public init(clientId: String, network: Network) {
+        self.clientId = clientId
+        self.network = network
+    }
+    
+    let clientId: String
+    let network: Network
+    var sdkURL: URL = URL(string: "https://sdk.openlogin.com")!
+}
+
+public struct OLLoginParams {
+    public init(provider: OpenLoginProvider? = nil, fastLogin: Bool? = nil, relogin: Bool? = nil, skipTKey: Bool? = nil, extraLoginOptions: Dictionary<String, Any>? = nil, redirectURL: String? = nil, appState: String? = nil) {
+        self.provider = provider
+        self.fastLogin = fastLogin
+        self.relogin = relogin
+        self.skipTKey = skipTKey
+        self.extraLoginOptions = extraLoginOptions
+        self.redirectURL = redirectURL
+        self.appState = appState
+    }
+     
+    public init(provider: OpenLoginProvider? = nil) {
+        self.provider = provider
+        self.fastLogin = nil
+        self.relogin = nil
+        self.skipTKey = nil
+        self.extraLoginOptions = nil
+        self.redirectURL = nil
+        self.appState = nil
+    }
+    
+    let provider: OpenLoginProvider?
+    let fastLogin: Bool?
```

Comment:
the casing seems differnet on `redirectUrl`. 
Pls follow the same as web for easier serialization
https://github.com/torusresearch/OpenLoginSdk/blob/master/packages/openlogin/src/constants.ts

Reply:
ok